docs/Overview.mdx

```diff
@@ -5,83 +5,77 @@ description: Casbin overview
-
-Feature                 | Go   | Java | Node.js | PHP  | Python | C#   | Delphi | Rust | C++ | Lua  |Dart  | Elixir
-:---                    | :--: | :--: | :--:    | :--: | :--:   | :--: | :--:   | :--: | :--:| :--: | :--: | :--: 
-Enforcement             | ✅   | ✅    | ✅      | ✅   | ✅      | ✅   | ✅     | ✅    | ✅  | ✅   | ✅    | ✅
-RBAC                    | ✅   | ✅    | ✅      | ✅   | ✅      | ✅   | ✅     | ✅    | ✅  | ✅   | ✅    | ✅
-ABAC                    | ✅   | ✅    | ✅      | ✅   | ✅      | ✅   | ✅     | ✅    | ✅  | ✅   | ✅    | ✅
-Scaling ABAC (`eval()`) | ✅   | ✅    | ✅      | ✅   | ✅      | ✅   | ❌     | ✅    | ✅  | ✅   | ✅    | ✅
-Adapter                 | ✅   | ✅    | ✅      | ✅   | ✅      | ✅   | ✅     | ✅    | ✅  | ✅   | ✅    | ❌
-Management API          | ✅   | ✅    | ✅      | ✅   | ✅      | ✅   | ✅     | ✅    | ✅  | ✅   | ✅    | ✅
-RBAC API                | ✅   | ✅    | ✅      | ✅   | ✅      | ✅   | ✅     | ✅    | ✅  | ✅   | ✅    | ✅
-Batch API               | ✅   | ✅    | ✅      | ✅   | ✅      | ✅   | ❌     | ✅    | ✅  | ✅   | ❌    | ❌
-Filtered Adapter        | ✅   | ✅    | ✅      | ✅   | ✅      | ✅   | ❌     | ✅    | ✅  | ✅   | ❌    | ❌
-Watcher                 | ✅   | ✅    | ✅      | ✅   | ✅      | ✅   | ✅     | ✅    | ✅  | ✅   | ❌    | ❌
-Role Manager            | ✅   | ✅    | ✅      | ✅   | ✅      | ✅   | ❌     | ✅    | ✅  | ✅   | ✅    | ❌
-Multi-Threading         | ✅   | ✅    | ✅      | ❌   | ✅      | ❌   | ❌     | ✅    | ❌  | ❌   | ❌    | ❌
-'in' of matcher         | ✅   | ✅    | ✅      | ✅   | ✅      | ❌   | ✅     | ❌    | ❌  | ❌   | ✅    | ✅
-
-**Note**- ✅ for Watcher or Role Manager 
-only means having the interface in the core library. It is not indicative of whether 
-there is a watcher or role manager implementation available.
+Enforcing a set of rules is as simple as listing subjects, objects and the desired allowed action (or any other format per your needs) in a policy file. This is synonymous across all flows Casbin is used in. The developer/administrator has complete control over the layout, execution and conditions for authorization which is set via the model file. Casbin provides an Enforcer for validating an incoming request based on the policy and model files given to the Enforcer.
 
 ## What is Casbin?
 
 Casbin is an authorization library which can be used in flows where we want a certain
-`object` or entity to be accessed by a specific user or `subject`. The type of access i.e. `action`
+`object` or entity to be accessed by a specific user or `subject`. The type of access (`action`)
 can be _read_, _write_, _delete_ or any other action as set by the developer.
 This is how Casbin is most widely used and its called the "standard"
 or classic `{ subject, object, action }` flow.
 
-Casbin is capable of handling many complex authorization scenarios other than the standard flow.
+Casbin is capable of handling many complex authorization senerios other than the standard flow.
```

Comment:
Why change to wrong one `senerios`? Better check all such mistakes

Reply:
That was an oversite, the necessary corrections have been made into the new commit.